advanced_source/cpp_export.rst

```diff
@@ -0,0 +1,366 @@
+Loading a PyTorch model in C++
+==============================
+
+As its name suggests, the primary interface to PyTorch is the Python
+programming language. While Python is a suitable and preferred language for
+many scenarios requiring dynamism and ease of iteration, there are equally many
+situations where precisely these properties of Python are unfavorable. One
+environment in which the latter often applies is *production* -- the land of
+low latencies and strict deployment requirements. For production scenarios, C++
+is very often the language of choice, even if only to bind it into another
+language like Java, Rust or Go. The following paragraphs will outline the path
+PyTorch provides in order to go from an existing Python model to a serialized
+representation that can be *loaded* and *executed* purely from C++ with no
+dependency on Python.
+
+Step 1: Converting Your PyTorch Model to Torch Script
+-----------------------------------------------------
+
+A PyTorch model's journey from Python to C++ is enabled by `Torch Script
+<https://pytorch.org/docs/master/jit.html>`_, a representation of a PyTorch
+model that can be understood, compiled and serialized by the Torch Script
+compiler. If you are starting out from an existing PyTorch model written in the
+vanilla "eager" API, you must first convert your model to Torch Script. In the
+most common cases, exemplified below, this requires only little effort. If you
+already have a *scripted* module (a module written in Torch Script), you can
+skip to the next section of this tutorial.
+
+There are two primary ways of converting a PyTorch model to Torch Script. The
+first is known as "tracing", a mechanism in which the structure of the model is
+captured by evaluating it once using examplary inputs, and recording the flow
```

Comment:
"examplary" -> "exemplary", although i think that has a different connotation than "example" (i.e. "exemplary" implies there exist desiderata to be copied, but "example" is just an instance of something). "Example inputs" sounds right to me